src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -15,6 +15,7 @@ import net.snowflake.spark.snowflake.TelemetryTypes.TelemetryTypes
 object SnowflakeTelemetry {
 
   private val TELEMETRY_SOURCE = "spark_connector"
+  private val MAX_OUTPUT_NODE_COUNT = 1024
```

Comment:
Just for my understanding, what is this?

Reply:
Telemetry service may send the logical plan to snowflake. And LogicalPlan.argString() is called to convert the plan as a String. From spark 3.0, this function needs a parameter: maxFields which can limit the fields count to be converted to a string. So from our side, we just provide a magic number like 1024 at the max count.